maxim/logger/livekit/utils.py

```
@@ -149,3 +152,34 @@ def start_new_turn(session_info: SessionStoreEntry):
             )
     return current_turn
+
+def extract_llm_usage(session_id: str, llm: Optional[LLM]) -> Optional[LLMUsage]:
+    if llm is None:
+        return None
+
+    if hasattr(llm, "_events") and llm._events is not None:
+        events = llm._events or getattr(llm, "_events", None)
+        if isinstance(events, dict) and "metrics_collected" in events:
+            session_info = get_session_store().get_session_by_agent_session_id(
+                session_id
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)